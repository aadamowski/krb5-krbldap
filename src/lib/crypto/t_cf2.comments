The first test  mirrors the first two tests in t_prf.in.

The second test mirrors the following four tests in t_prf.in.
