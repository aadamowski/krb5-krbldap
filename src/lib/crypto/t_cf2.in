17
key1
key2
a
b
18
key1
key2
a
b
