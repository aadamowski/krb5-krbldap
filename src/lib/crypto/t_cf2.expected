97df97e4b798b29eb31ed7280287a92a
4d6ca4e629785c1f01baf55e2e548566b9617ae3a96868c337cb93b5e72b1c7b
